path.go

```diff
@@ -71,6 +72,18 @@ func getFinalRecord(zone string, from int) (record, error) {
 		return rec, fmt.Errorf("could not parse record: %s", err)
 	}
 
+	if rec.Type == "path" {
+		url, err := url.Parse(rec.To)
+		if err != nil {
+			return rec, fmt.Errorf("could not parse the url: %s", err)
+		}
+		zone, from, err := zoneFromPath(url.Host, url.Path, rec)
+		if err != nil {
+			return rec, fmt.Errorf("could not generate zone URI from path. %s", err)
```

Comment:
Small consistency issue with the error message following a period `.` instead of a colon `:` like in other error messages of the file